src/SOS/Strike/sos.def

```diff
@@ -18,8 +18,6 @@ EXPORTS
     dumparray=DumpArray
     DumpAssembly
     dumpassembly=DumpAssembly
-    DumpAsync
-    dumpasync=DumpAsync
```

Comment:
If you leave these here and set a stub like https://github.com/dotnet/diagnostics/blob/55a08370317a26e4d1c2b071936d6e93632b4c45/src/SOS/Strike/strike.cpp#L16871-L16878 and the same for the lldb extension then we could support this command as a first class citizen. This makes sense since it's a widely applicable command that was already working without the `ext` prefix.

Reply:
Done